### PR TITLE
Using Apphost.exe for standalone apps

### PR DESCRIPTION
**Pull request by ramarag**
@eerhardt @nguerrera @livarcocc @gkhanna79  PTAL
This is a followup of https://github.com/dotnet/sdk/pull/978 towards

### CODE REVIEW

File: build/DependencyVersions.props [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <CLI_SharedFrameworkVersion>2.0.0-beta-001791-00</CLI_SharedFrameworkVersion>
+    <CLI_SharedFrameworkVersion>2.0.0-beta-001783-00</CLI_SharedFrameworkVersion>
```
Comment (livarcocc) [inline build/DependencyVersions.props]: Why are we going back on the shared framework version?

Comment (ramarag) [inline build/DependencyVersions.props]: Oops, will change it back

**Comment by livarcocc** [comment]
@dotnet-bot Test Fedora24 x64 Release Build
@dotnet-bot Test OpenSUSE42.1 x64 Debug Build

**Comment by ramarag** [comment]
@dotnet-bot test Ubuntu16.10 x64 Debug Build

**Pull request merged**